build/constants.yaml

```diff
@@ -216,6 +216,7 @@
     bash: /bin/bash
     benv: /opt/oryx/benv
     pre-run-command-env-var-name: PRE_RUN_COMMAND
+    app-svc-file-name: appsvc.yaml
```

Comment:
`generateConstants.sh` was overwriting a go file that removed this constant that is needed for the startupscriptgenerator